kibicara/platforms/email/model.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2020 by Maike <tom@dl6tom.de>
+#
+# SPDX-License-Identifier: 0BSD
+
+from kibicara.model import Hood, Mapping
+from ormantic import Integer, ForeignKey, Model, Text
+
+
+class EmailSubscribers(Model):
+    """ This table stores all subscribers, who want to receive messages via email. """
+
+    id: Integer(primary_key=True) = None
+    hood: ForeignKey(Hood)
```

Comment:
depending on `Email` instead of `Hood` would make it more obvious that there should be no subscribers if there is no entry in `Email` (and may allow cascading deletions on the database level later).